README.md

```diff
@@ -24,6 +24,10 @@ The Espresso package consists of two main parts:
 The key difference between [UiAutomator2 Driver](https://github.com/appium/appium-uiautomator2-driver) and Espresso Driver is that UiAutomator2 is a black-box testing framework, and Espresso is a "grey-box" testing framework. The Espresso Driver itself is black-box (no internals of the code are exposed to the tester), but the Espresso framework itself has access to the internals of Android applications. This distinction has a few notable benefits. It can find elements that aren't rendered on the screen, it can identify elements by the Android View Tag and it makes use of [IdlingResource](https://developer.android.com/reference/android/support/test/espresso/IdlingResource) which blocks the framework from running commands until the UI thread is free. There is limited support to automate out of app areas using the mobile command [uiautomator](https://github.com/appium/appium-espresso-driver/blob/b2b0883ab088a131a47d88f6aeddd8ff5882087d/lib/commands/general.js#L188)
 
 
+## Jetpack compose dependencies
+
+Espresso driver [suppots Jetpack Compose](#jetpack-compose-support) by default while the gradle dependencies affect badly for the application under test in some cases. For example, the application under test does have the Jetpack Compose dependencies. Then. you can try out [no compose dependencies branch](https://github.com/appium/appium-espresso-driver/pull/879)). In Appium 2.0, the branc is available as `appium driver install --source=local /path/to/the/appium-espress-driver` with the `no-compose-deps` branch instead of npm installation.
```

Comment:
branc  -> branch

Reply:
> by default while the gradle dependencies affect badly for the application under test in some cases

may it be rephrased? I cannot get the meaning of this sentence

Reply:
Tweaked as https://github.com/appium/appium-espresso-driver/pull/880/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5